tonic/Cargo.toml

```diff
@@ -39,6 +41,13 @@ tls-roots-common = ["tls"]
 tls-roots = ["tls-roots-common", "rustls-native-certs"]
 tls-webpki-roots = ["tls-roots-common", "webpki-roots"]
 prost = ["prost1", "prost-derive"]
+client = [
```

Comment:
should this feature be called `wasm`?

Reply:
Technically, this is not about `wasm` but about having a client without relying on any OS features e.g. like sockets. But naming is hard and I agree `client` is somehow misleading.